blueprints/screening_app/domains/prediabetes_screening/opt_in/train.txt

```diff
@@ -0,0 +1,1641 @@
+{2|sys_number} {metros|unit} y {1|sys_number} {centimetro|unit} de altura
+{2|sys_number} {metros|unit} y {1|sys_number} {centimetro|unit} de estatura
+Mido {2|sys_number}
+Mi estatura es {2|sys_number}
+Mi altura es {2|sys_number}
+Tengo {2|sys_number} de altura
+Tengo {2|sys_number} de estatura
+Mido {2|sys_number} de altura
+Mido {2|sys_number} de estatura
+Yo mido {2|sys_number} de altura
+Yo mido {2|sys_number} de estatura
+Yo tengo {2|sys_number} de altura
+Yo tengo {2|sys_number} de estatura
+Mido {2|sys_number} {metros|unit}
+Mi estatura es {2|sys_number} {metros|unit}
+Mi altura es {2|sys_number} {metros|unit}
+Yo mido {2|sys_number} {metros|unit}
+Tengo {2|sys_number} {metros|unit} de altura
+Tengo {2|sys_number} {metros|unit} de estatura
+Mido {2|sys_number} {metros|unit} de altura
+Mido {2|sys_number} {metros|unit} de estatura
+Yo mido {2|sys_number} {metros|unit} de altura
+Yo mido {2|sys_number} {metros|unit} de estatura
+Yo tengo {2|sys_number} {metros|unit} de altura
+Yo tengo {2|sys_number} {metros|unit} de estatura
+Mido {2|sys_number} {metros|unit} {1|sys_number} {centimetro|unit}
+Mi estatura es {2|sys_number} {metros|unit} {1|sys_number} {centimetro|unit}
+Mi altura es {2|sys_number} {metros|unit} {1|sys_number} {centimetro|unit}
+Yo mido {2|sys_number} {metros|unit} {1|sys_number} {centimetro|unit}
+Tengo {2|sys_number} {metros|unit} {1|sys_number} {centimetro|unit} de altura
+Tengo {2|sys_number} {metros|unit} {1|sys_number} {centimetro|unit} de estatura
+Mido {2|sys_number} {metros|unit} {1|sys_number} {centimetro|unit} de altura
+Mido {2|sys_number} {metros|unit} {1|sys_number} {centimetro|unit} de estatura
+Yo mido {2|sys_number} {metros|unit} {1|sys_number} {centimetro|unit} de altura
+Yo mido {2|sys_number} {metros|unit} {1|sys_number} {centimetro|unit} de estatura
+Yo tengo {2|sys_number} {metros|unit} {1|sys_number} {centimetro|unit} de altura
+Yo tengo {2|sys_number} {metros|unit} {1|sys_number} {centimetro|unit} de estatura
+Mido {2|sys_number} {metros|unit} y {1|sys_number} {centimetro|unit}
+Mi estatura es {2|sys_number} {metros|unit} y {1|sys_number} {centimetro|unit}
+Mi altura es {2|sys_number} {metros|unit} y {1|sys_number} {centimetro|unit}
+Yo mido {2|sys_number} {metros|unit} y {1|sys_number} {centimetro|unit}
+Tengo {2|sys_number} {metros|unit} y {1|sys_number} {centimetro|unit} de altura
+Tengo {2|sys_number} {metros|unit} y {1|sys_number} {centimetro|unit} de estatura
+Mido {2|sys_number} {metros|unit} y {1|sys_number} {centimetro|unit} de altura
+Mido {2|sys_number} {metros|unit} y {1|sys_number} {centimetro|unit} de estatura
+Yo mido {2|sys_number} {metros|unit} y {1|sys_number} {centimetro|unit} de altura
+Yo mido {2|sys_number} {metros|unit} y {1|sys_number} {centimetro|unit} de estatura
+Yo tengo {2|sys_number} {metros|unit} y {1|sys_number} {centimetro|unit} de altura
+Yo tengo {2|sys_number} {metros|unit} y {1|sys_number} {centimetro|unit} de estatura
```

Comment:
@ritvikshrivastava can the form use this kind of complexity? it would make the experience less robotic.

Reply:
Do you mean multiple entities at once? 

If the initial query has multiple entities (like the examples here), they get pre-populated in the form and the user isn't prompted for them.

Reply:
This is later in the form, in response to the question "how tall are you?", answering "I am {2|sys_number} {meters|unit} and {1|sys_number} {centimeters|unit} tall". {2 meters} and {1 centimeter} are known distance entities in English, but not in Spanish, and the combined phrase is not recognized in either language.

Reply:
Hmm, I'll look if that initial fill function can be used in between here. 

Alternatively, one direct way currently could be to prompt the message only asking the weight/height in a specific unit. say Kilos/Metres? The app logic can then be calculated using that selected unit always as the onus on replying to the prompt with that unit is on the user.

Reply:
Can be resolved using https://github.com/lfiedlerc/mindmeld-blueprints/pull/1 and https://github.com/cisco/mindmeld/pull/205.